fast-list.js

```diff
@@ -112,7 +117,7 @@ FastList.prototype = {
     var itemPerScreen = geo.viewportHeight / geo.itemHeight;
     // Taking into account the will-change budget multiplier from
     // layout/base/nsDisplayList.cpp#1193
-    geo.maxItemCount = Math.floor(itemPerScreen * 2.8);
+    geo.maxItemCount = Math.floor(itemPerScreen * 3);
```

Comment:
well it's 1 more in practice and outside of the viewport since we're not scrollable at that point, so I'm not sure it'll show up :)

mainly I have 0 justification to choose between 80% of the budget or 90% of the budget so I'd rather just cleanly align with the budget value.